### PR TITLE
New Readme Design

### PR DESCRIPTION
**Pull request by vihar**
New Readme Design

- New Hero Image
- Added all Social Links
- Update Content
- Update Images and GIFs
- Add New Features
- Updated Demo's and Tutorials
- New Contributors List Design

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,175 +1,158 @@
...
 
-- [Customer Support Dashboard](https://bit.ly/cs-dashboard-appsmith)
-- [Job Application Tracker](https://bit.ly/3hbYtTi)
+> _Currently works with PostgreSQL, MongoDB, MySQL, Firestore, S3, Redshift, Elastic Search, DynamoDB, Redis, and MSFT SQL Server_ > <br>
 
-## 🏃‍♀️ Quickstart
+#### 🤖 Code Logic with JavaScript
 
-The following steps introduce you to building a simple user-list dashboard on Appsmith.
+All parts of your app can be referenced as JavaScript objects. Code interactions between elements, update datasources or manipulate SQL queries—[we'll parse JS anywhere](https://docs.appsmith.com/core-concepts/writing-code).<br><br>
```
Comment (Nikhil-Nandagopal) [inline README.md:58]: Change to "we'll run JS anywhere"

File: README.md [hunk 2]
```diff
@@ -1,175 +1,158 @@
...
-- Tutorial Videos: [Youtube](https://www.youtube.com/appsmith)
-- Community Support: [Discord](https://discord.gg/rBTTVJp)
-- Issue & bug tracking: [GitHub Issues](https://github.com/appsmithorg/appsmith/issues/new/choose)
+## Demos
 
-## 🧑‍🤝‍🧑 Contributing
+Check out some real applications that can be built with Appsmith:
 
-If you're interested in contributing to Appsmith:
+- 👩‍💻 [Customer Support Dashboard](https://app.appsmith.com/applications/5f2aeb2580ca1f6faaed4e4a/pages/5f2d61b580ca1f6faaed4e79/?utm_source=github&utm_medium=direct&utm_content=appsmith_apps&utm_campaign=null&utm_term=appsmith_apps)
```
Comment (Nikhil-Nandagopal) [inline README.md:81]: Can we add the other templates at appsmith.com/templates

File: README.md [hunk 3]
```diff
@@ -1,175 +1,158 @@
...
 
-We are committed to fostering an open and welcoming environment in the community. Please read our [Code of Conduct](CODE_OF_CONDUCT.md).
+Here are the latest tutorials and videos for you to learn more about Appsmith:
+
+1. [Building an Admin Panel on MongoDB using Appsmith](https://blog.appsmith.com/building-an-admin-panel-with-mongodb-using-appsmith) ([Video](https://www.youtube.com/watch?v=tisUaIgI86k))
+2. [Building a Customer Support Dashboard in Appsmith](https://www.youtube.com/watch?v=-O_6OLREEzo)
+3. [Building a Store Catalogue Management System using Appsmith and GraphQL](https://blog.appsmith.com/building-a-store-catalogue-management-system-using-appsmith-and-graphql)
+4. [Running CI/CD Jobs Manually using Appsmith](https://blog.appsmith.com/how-to-run-manual-jobs-in-gitlab-cicd) ([Video](https://www.youtube.com/watch?v=CYdeJcD4I8A))
+5. [Building a Calendly Clone in Appsmith](https://blog.appsmith.com/how-to-build-a-calendly-clone-in-30-minutes)
+6. [Building Internal Tools with Appsmith](https://youtu.be/eYYYfuW-kEE) `Community`
```
Comment (Nikhil-Nandagopal) [inline README.md:95]: what does the community tag signify here?

File: README.md [hunk 4]
```diff
@@ -1,175 +1,158 @@
...
+## Features
 
-<img src="https://github.com/appsmithOrg/appsmith/blob/release/static/UI.gif">
-<p align="center">
-  <img src="https://github.com/appsmithOrg/appsmith/blob/release/static/Query2.png" width="270">
-  <img src="https://github.com/appsmithOrg/appsmith/blob/release/static/API2.png" width="270">
-  <img src="https://github.com/appsmithOrg/appsmith/blob/release/static/Share5.png" width="270">
+<p>
+  <img src="static/images/animations/drag-drop.gif" width="32%">
+  <img src="static/images/animations/selections.gif" width="32%">
```
Comment (Nikhil-Nandagopal) [inline README.md:40]: All 3 images are doing the same thing. Can we get one around writing JS and one around running an API

**Review by Nikhil-Nandagopal — CHANGES_REQUESTED** [review]
@vihar @akshayrangasaid I've suggested a few changes

File: README.md (hunk 3 above)
Comment (vihar) [reply README.md:95]: We add a `Community` tag when our Community creates the content. These are published on their own blog/youtube channels, not on Appsmith websites.

File: README.md (hunk 4 above)
Comment (vihar) [reply README.md:40]: Updated, thanks for the suggestion!

**Comment by vihar** [comment]
@akshayrangasaid and @Nikhil-Nandagopal, I've updated based on the feedback. Let me know if we have to make any additional changes.

**Review by Nikhil-Nandagopal — APPROVED** [review]

**Review by akshayrangasaid — APPROVED** [review]

**Review by Nikhil-Nandagopal — APPROVED** [review]

**Pull request merged**